configuration.example.yaml

```diff
@@ -48,8 +48,11 @@ aws:
   enabled: false
   # The endpoint of the WebSocket API
   endpoint: wss://abcdef1234.execute-api.eu-central-1.amazonaws.com/production
-  # Currently, only static AWS environments are supported. You can list them here separated by spaces.
-  functions: "java11Exec go118Exec"
+  # Currently, only static AWS environments are supported.
+  # For setting this via environment variables you have to use a string separated by spaces, like: POSEIDON_AWS_FUNCTIONS="java11Exec go118Exec".
```

Comment:
I assume this comment is now outdated, isn't it? 🤓

Reply:
Hm, I guess not. Since [`The value of an environment variable is a string of characters`](https://pubs.opengroup.org/onlinepubs/9699919799/basedefs/V1_chap08.html) the option to set this value as environment variable (not via the config file! Like it is used in our CI) must still rely on a string separated by any kind of delimiter..

This PR just fixes that our yaml config file is parsing a list from a string although lists can be specified in yaml.

Reply:
Ah, I see. 🙈 Thanks for reminding me 😄